libraries/Microsoft.Bot.Connector/Authentication/JwtTokenValidation.cs

```diff
@@ -108,7 +109,7 @@ public static async Task<ClaimsIdentity> ValidateAuthHeader(string authHeader, I
         /// identity for the request.</remarks>
         public static async Task<ClaimsIdentity> ValidateAuthHeader(string authHeader, ICredentialProvider credentials, IChannelProvider channelProvider, string channelId, AuthenticationConfiguration authConfig, string serviceUrl = null, HttpClient httpClient = null)
         {
-            if (string.IsNullOrEmpty(authHeader))
+            if (String.IsNullOrEmpty(authHeader))
```

Comment:
fat fingers, reverted it.

BTW, in some cases we are starting to use string.IsNullOrWhitespace instead of IsNullOrEmpty, not sure if it applies here, what do you think?